app/react-graphql/hooks/useMutate.utils.tsx

```diff
@@ -1,71 +1,101 @@
-import { getFieldFragmentInfo } from '../support/HasuraConfigUtils';
-import { print } from 'graphql';
+import {getFieldFragmentInfo} from '../support/HasuraConfigUtils';
```

Comment:
Nothing of import changed in here.  I just bumped my format button and tweaked some logging and minor cleanup on the delete util to mirror other fixes in the other two.